youtube_base/youtubers/feeds.py

```diff
@@ -0,0 +1,49 @@
+import os
+import markdown
+from django.contrib.sites.models import Site
+from django.contrib.syndication.views import Feed
+from django.template.defaultfilters import truncatewords_html
+from django.urls import reverse_lazy
+
+from .models import Youtuber
+
+
+class LatestYoutubersFeed(Feed):
+    """A Feed class that generates an RSS feed for the latest Youtubers."""
+    title = 'Українська база ютуберів.'
+    desription = 'Новий ютубер у нашій базі.'
+
+    def items(self):
+        """Return the Youtuber objects that should be included in the feed."""
+        return Youtuber.objects.all()[:5]
```

Comment:
why do we take only five first objects?

let's invent a constant for this magic variable.